resources/migrations/001_update_migrations.yaml

```diff
@@ -7364,6 +7364,73 @@ databaseChangeLog:
         - customChange:
             class: "metabase.db.custom_migrations.MigrateStackedAreaBarComboDisplaySettings"
 
+  - changeSet:
+      id: v50.2024-05-17T19:54:23
+      author: calherries
+      comment: Add metabase_database.uploads_enabled column
+      changes:
+        - addColumn:
+            tableName: metabase_database
+            columns:
+              - column:
+                  name: uploads_enabled
+                  type: ${boolean.type}
+                  defaultValueBoolean: false
+                  remarks: Whether uploads are enabled for this database
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: v50.2024-05-17T19:54:24
+      author: calherries
+      comment: Add metabase_database.uploads_schema_name column
+      changes:
+        - addColumn:
+            tableName: metabase_database
+            columns:
+              - column:
+                  name: uploads_schema_name
+                  type: ${text.type}
+                  remarks: The schema name for uploads
+
+  - changeSet:
+      id: v50.2024-05-17T19:54:25
+      author: calherries
+      comment: Add metabase_database.uploads_table_prefix column
+      changes:
+        - addColumn:
+            tableName: metabase_database
+            columns:
+              - column:
+                  name: uploads_table_prefix
+                  type: ${text.type}
+                  remarks: The prefix for upload table names
+
+  - changeSet:
+      id: v50.2024-05-17T19:54:26
+      author: calherries
+      comment: Update metabase_database.uploads_enabled value
+      changes:
+        - customChange:
+            class: "metabase.db.custom_migrations.MigrateUploadsSettings"
+
+  - changeSet:
+      id: v50.2024-05-17T19:54:27
+      author: calherries
+      comment: Remove uploads settings
+      changes:
+        - sql:
```

Comment:
Why not do this clean-up in the custom migration instead? That avoids the quirk of having the rollback in a previous migration, and toucan2 can paper over the dialect differences.

Reply:
no good reason, I had just written this SQL migration first. I've tidied it up as you suggest here 33697533e327a3593622254371d01b1ae0dfe563